packages/api-client/tests/test-collections.js

```diff
@@ -1,12 +1,35 @@
 'use strict';
 
 const test = require('ava');
+
+const {
+  createBucket,
+  recursivelyDeleteS3Bucket,
+} = require('@cumulus/aws-client/S3');
+
 const collectionsApi = require('../collections');
+const { fakeCollectionFactory } = require('../../api/lib/testUtils');
+const { Collection } = require('../../api/models');
+const { randomId, randomString } = require('../../common/test-utils');
 
-test.before((t) => {
+test.before(async (t) => {
+  process.env.stackName = randomString();
+  process.env.system_bucket = randomString();
   t.context.testPrefix = 'unitTestStack';
   t.context.collectionName = 'testCollection';
   t.context.collectionVersion = 1;
+
+  // create a fake bucket
+  await createBucket(process.env.system_bucket);
+
+  process.env.CollectionsTable = randomId('collection');
+  t.context.collectionModel = new Collection();
+  await t.context.collectionModel.createTable();
```

Comment:
You don't need this now

Reply:
[`eb916aa` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/eb916aa52b4a766b18602366fbc3550eb2deb9ae)